test/libsolidity/SolidityParser.cpp

```diff
@@ -120,8 +120,9 @@ BOOST_AUTO_TEST_CASE(reserved_keywords)
 {
 	BOOST_CHECK(!TokenTraits::isReservedKeyword(Token::Identifier));
 	BOOST_CHECK(TokenTraits::isReservedKeyword(Token::After));
-	BOOST_CHECK(TokenTraits::isReservedKeyword(Token::Unchecked));
+	BOOST_CHECK(!TokenTraits::isReservedKeyword(Token::Unchecked));
 	BOOST_CHECK(TokenTraits::isReservedKeyword(Token::Var));
+	BOOST_CHECK(TokenTraits::isReservedKeyword(Token::Reference));
```

Comment:
Where did `Reference` come from?

Reply:
IIUC the test verifies that ``isReservedKeyword`` is correctly true for the first and the last reserved keyword - the last one used to be ``unchecked``, but then we made ``var`` reserved again and added it as new last one here, but just kept ``unchecked``, so since then the idea is "check first and last reserved keyword and a random one in between" - the random one in between can't be unchecked anymore, so now it's reference...